org_status/status_providers/appveyor.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+from org_status.status_providers import StatusProvider, Status
+
+
+class AppveyorBuildStatus(StatusProvider):
+    BadgeTemplate = ('https://ci.appveyor.com/api/projects/status/github/'
```

Comment:
The status providers are not automatically called yet.

I'll upload https://github.com/ksdme/org-status/issues/17 and a few others, after this.

So you will need to manually invoke both Travis and AppVeyor status providers from the GitHub org host.

AppVeyor is also available on GitLab, so ideally you also add AppVeyor to the GitLab results, which means the literal 'github' here will need to become generic.